src/main/java/io/codeka/gaia/client/controller/BuildInfoRestController.kt

```diff
@@ -0,0 +1,26 @@
+package io.codeka.gaia.client.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.info.BuildProperties
+import org.springframework.boot.info.GitProperties
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/build-info")
+class BuildInfoRestController constructor(
```

Comment:
You can remove the `constructor` keyword here

Reply:
:heavy_check_mark: